datalake/backend/processing/import_dataset/task.py

```diff
@@ -0,0 +1,142 @@
+import json
+import logging
+from datetime import datetime
+from http.client import responses as http_responses
+from os import environ
+from typing import Any, List, MutableMapping, Union
+from urllib.parse import urlparse
+from uuid import uuid4
+
+import boto3
+from jsonschema import ValidationError, validate  # type: ignore[import]
+from smart_open import open as smart_open  # type: ignore[import]
+
+from ..assets_model import ProcessingAssetsModel
+
+JsonList = List[Any]
+JsonObject = MutableMapping[str, Any]
+
+ssm_client = boto3.client("ssm")
+sts_client = boto3.client("sts")
+s3_client = boto3.client("s3")
+s3control_client = boto3.client("s3control")
+
+ENV = environ.get("DEPLOY_ENV", "test")
+STORAGE_BUCKET_PARAMETER = f"/{ENV}/storage-bucket-arn"
+S3_BATCH_COPY_ROLE_PARAMETER = f"/{ENV}/s3-batch-copy-role-arn"
```

Comment:
These should probably be have a `_NAME` suffix to distinguish the name from the value.